xmtp_mls/src/groups/mod.rs

```diff
@@ -378,6 +382,19 @@ where
         self.sync_until_intent_resolved(conn, intent.id).await
     }
 
+    pub async fn update_group_metadata(&self, group_name: String) -> Result<(), GroupError> {
```

Comment:
If all this function does is update the group name, maybe we should rename it to be more specific. Otherwise we'll have a breaking change when we add a second metadata field.

Reply:
nice catch 🙏 - updated here: f2f1a966a0be2c8b8b1d4ee8450a86a6f201abd4